src/UI/Actions.hs

```diff
@@ -1321,6 +1322,19 @@ fileBrowserToggleFile =
     fb' <- lift $ FB.maybeSelectCurrentEntry fb
     assign (asFileBrowser . fbEntries) fb'
 
+-- | Search related mails sent from the same authors.
+--
+searchRelated :: Action 'Threads 'ListOfThreads ()
+searchRelated = Action ["search related mail"] $ do
+  authors <- gets (concatOf (asThreadsView . miThreads . listList . to L.listSelectedElement . _Just . to snd . to snd . thAuthors))
+  case firstOf traverse authors of
```

Comment:
I think this can be written as:
```haskell
authors <- preuse (asThreadsView . miThreads . listList . to L.listSelectedElement . _Just . _2 . _2 . thAuthors . traverse)
```

Reply:
Indeed!! Thanks @frasertweedale-hlint ;)